packages/create-next-app/create-app.ts

```diff
@@ -132,7 +131,7 @@ export async function createApp({
 
   const appName = path.basename(root)
 
-  await makeDir(root)
+  fs.mkdirSync(root, { recursive: true })
```

Comment:
Any reason this was changed from async to sync?

Reply:
@styfle `create-next-app` is just a simple single-thread script-like app that doesn't have multiple concurrent promises.

I've explained that before: https://github.com/vercel/next.js/pull/56917#discussion_r1362421015